### PR TITLE
Added new plugins to the plugin list

### PR DESCRIPTION
**Pull request by sandlbn**
Fixes #

Summary of changes:
- Added 2 new plugins from Intel Labs to the list of plugins

Testing done:
- Not needed

@intelsdi-x/snap-maintainers

### CODE REVIEW

File: docs/PLUGIN_CATALOG.md [hunk 1]
```diff
@@ -59,7 +59,7 @@ These plugins are in planned/active development. This list is useful if you want
 | Cassandra | Collector | Collects metrics from Cassandra cluster | [snap-plugin-collector-cassandra](https://github.com/intelsdi-x/snap-plugin-collector-cassandra) | [@candysmurf](https://github.com/candysmurf) |
 | Open vSwitch | Collector | Collect Open vSwitch performance data | -| [@sandlbn](https://github.com/sandlbn) |
 | Cassandra | Publisher | Publishes snap metrics into Cassandra | [snap-plugin-publisher-cassandra](https://github.com/intelsdi-x/snap-plugin-publisher-cassandra) | [@candysmurf](https://github.com/candysmurf) |
-
+| OpenFoam | Collector | Collect metrics from OpenFoam | [snap-plugin-collector-openfoam](https://github.com/intelsdi-x/snap-plugin-collector-openfoam) | [@sandlbn](https://github.com/sandlbn) |
```
Comment (candysmurf) [inline docs/PLUGIN_CATALOG.md]: link seems not reachable.

Comment (tiffanyfj) [inline docs/PLUGIN_CATALOG.md]: Yeah, I think we don't have the permissions to see that repo.

**Comment by mjbrender** [comment]
@sandlbn could you add some description to the README.md and then open this up from private to public before linking? 

Keep these plugins and doc updates coming 👍

**Comment by sandlbn** [comment]
@mjbrender ok done. It's open now.

**Comment by tiffanyfj** [comment]
👍

**Pull request merged**